companion/src/eepromimportexport.h

```diff
@@ -698,8 +698,11 @@ class ConversionField: public TransformedField {
       if (table) {
         if (table->exportValue(_field, _field))
           return;
-        if (!error.isEmpty())
+        if (!error.isEmpty()) {
+          // TODO how could we avoid this global?
```

Comment:
well, I am not convinced :) I would like to add a parent to each field so that `setError(...)` calls `parent->setError(...)` but I am afraid of having far too many changes ... well perhaps not an issue